spec/unit/oidc/tokenRefresher.spec.ts

```diff
@@ -38,22 +38,8 @@ describe("TokenRefresher()", () => {
         sub: "123",
         iat: 123,
     };
+    // used to mock a valid token response, as consumed by OidcClient library
```

Comment:
```suggestion
    // a mocked valid token response, as consumed by OidcClient library
```

Reply:
the scope is *used* in the valid token response

Reply:
```suggestion
    // the `scope` part of a valid token response, as consumed by OidcClient library
```

then?